### PR TITLE
Cherisha3test

### PR DESCRIPTION
**Pull request by QiruJin**
# Pull Request Template

## Description of Test Case(s)

Please list the test case(s) that you create and corresponding functions. Please also include relevant motivation and context. 

## Test Case create for:
- [ ] Landing Session
- [ ] Login Session
- [ ] Logout Session
- [ ] Register Session
- [ ] Buy Session
- [ ] Sell Session
- [ ] Update Session
- [ ] Exit Session

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Test
- [ ] Integration Test
- [ ] System Test

## Test Environment:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

##### Local:

- [ ] MacOS
- [ ] Windows

##### IDLE environment:

- [ ] Pycharm
- [ ] VScode
- [ ] PythonIDLE
- [ ] Others

## Test Case Passed or not:

- [ ] Passed
- [ ] Fail

## Description of Source Code

If the source code has been changed. Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes code in sell, buy, update sessions. 
Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] More Details about dd comment & change text document

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by QiruJin** [comment]
After Merge, relative test cases should be failed in action and we need to rewrite several cases

File: qa327/frontend/sessions.py [hunk 1]
```diff
@@ -201,26 +201,27 @@ class UpdateSession(LoggedInSession):
...
             print(str(e))
             print('\nUpdate failed, ending session...')
         except exceptions.WrongTicketNameException:
             print('\nThe ticket name you entered cannot be found, ending session...')
 
+    def updateTicket(self, ticket_name, ticket_price, ticket_quantity, ticket_date):
+        helpers.TransactionsHelper.newTicketTransaction("update", self.username, ticket_name, ticket_price, ticket_quantity, ticket_date)
+        helpers.ResourcesHelper.getTicketInfo()[ticket_name]['price'] = ticket_price
+        helpers.ResourcesHelper.getTicketInfo()[ticket_name]['number'] = ticket_quantity
+        helpers.ResourcesHelper.getTicketInfo()[ticket_name]['date'] = ticket_date
```
Comment (rickyzhangca) [inline qa327/frontend/sessions.py:223]: why do you need these lines when you have passed everything in with line 220?

**Comment by rickyzhangca** [comment]
solved the conflict

**Review by rickyzhangca — APPROVED** [review]
check comment

File: qa327/frontend/sessions.py (hunk 1 above)
Comment (QiruJin) [reply qa327/frontend/sessions.py:223]: > solved the conflict

Are you the smartest boy in the world? and the answer is YES!💥

Comment (QiruJin) [reply qa327/frontend/sessions.py:223]: > why do you need these lines when you have passed everything in with line 220?

ASK ASK Cherish or Steven✊

**Review by cherishfan0320 — APPROVED** [review]

**Review by chengzhen-guo — APPROVED** [review]

**Pull request merged**